### DISCUSSION TITLE
EP sticky ?

### DISCUSSION THREAD

**Discussion by milsabords** [valinet/ExplorerPatcher#2388] [General]
I uninstalled EP using Revo, rebooted,  it does not show up in installed software, but EP is still there in Task Bar Properties.

**Comment by Amrsatrio** [comment]
Why didn't you uninstall EP normally via installed software?

**Comment by milsabords** [comment]
Revo first launches the native uninstaller, then finds and removes leftovers.

**Comment by pyrates999** [comment]
Try installing EP and then using the normal way to uninstall EP without using revo.  Lets see if that works.

**Reply by milsabords** [reply]
I will do that in case of problems, right now I use V226212428.59.1, no issue.
The reason I wanted to remove EP: I suspected that it caused  the failed install of 23H2 preview, which was not.
I use Open Shell and EP.

**Reply by pyrates999** [reply]
You should always uninstall EP before installing a new version of windows.
